CliTester: Nice error when cannot run PHP interpreter

xmedeko opened the pull request:
- feature
- BC break - no

dg commented on src/Runner/CliTester.php:
```diff
@@ -161,6 +161,9 @@ private function createPhpInterpreter()
 			NULL,
 			array('bypass_shell' => TRUE)
 		);
+		if ($proc === false) {
```
Use please `if ($proc === FALSE)` or `if (!$proc)`.

dg commented on src/Runner/CliTester.php:
```diff
@@ -161,6 +161,9 @@ private function createPhpInterpreter()
 			array('bypass_shell' => TRUE)
 		);
+		if ($proc === false) {
+			throw new \Exception('Cannot run php interpreter ' . $this->options['-p'] . '. Use -p option.');
```
PHP should be uppercased

Majkl578 commented on src/Runner/CliTester.php:
```diff
@@ -161,6 +161,9 @@ private function createPhpInterpreter()
 		);
+		if ($proc === false) {
+			throw new \Exception('Cannot run php interpreter ' . $this->options['-p'] . '. Use -p option.');
+		};
```
Extra semicolon.

xmedeko commented:
Hi guys, I don't know how to corrent the pull request since I've deleted my fork already. I have restored the fork from pull request commit and corrected it, see https://github.com/xmedeko/tester/commit/3a36426099ac3ae1385b5c83556f9317e1ae2002 or squashed into one correct commit see https://github.com/xmedeko/tester/commit/45a884e274db5af6640afd383a5c342f21ba501a
